src/lib/holocene/markdown.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+  import type { HTMLTextareaAttributes } from 'svelte/elements';
+
+  import { twMerge as merge } from 'tailwind-merge';
+
+  import Label from './label.svelte';
+
+  type $$Props = HTMLTextareaAttributes & {
+    disabled?: boolean;
+    placeholder?: string;
+    rows?: number;
+    spellcheck?: boolean;
+    value: string;
+    label: string;
+    labelHidden?: boolean;
+    id: string;
+    required?: boolean;
+    description?: string;
+    maxLength?: number;
+  };
+
+  export let disabled = false;
+  export let placeholder = '';
+  export let rows = 10;
+  export let spellcheck: boolean = null;
+  export let value: string;
+  export let label: string;
+  export let labelHidden = false;
+  export let id: string;
+  export let required = false;
+  export let description = '';
+  export let maxLength = 0;
+
+  let className = 'text-primary';
+  export { className as class };
+</script>
+
+<div class={className}>
+  <Label {required} hidden={labelHidden} {label} for={id} />
+  {#if description}
+    <p class="pb-2 text-sm">{description}</p>
+  {/if}
+  <div class="relative mt-2">
+    <textarea
+      bind:value
+      class={merge(
+        'surface-input min-h-fit w-full rounded-lg border-2 border-subtle px-3 py-2 font-mono text-sm focus-visible:border-information focus-visible:shadow-[0_0_0_4px_rgb(97,115,243,0.7)] focus-visible:outline-none enabled:hover:border-information',
```

Comment:
Follow up task - create a tailwind util for this arbitrary shadow class: https://tailwindcss.com/docs/box-shadow#customizing-your-theme

Reply:
Most certainly going to delete this anyways when we get proper Markdown